packages/headless/doc-parser/use-cases/insight.ts

```diff
@@ -96,18 +96,39 @@ const actionLoaders: ActionLoaderConfiguration[] = [
   {
     initializer: 'loadInsightSearchActions',
   },
+  {
+    initializer: 'loadQuerySetActions',
+  },
   {
     initializer: 'loadInsightAnalyticsActions',
   },
   {
-    initializer: 'loadInsightSearchAnalyticsActions',
```

Comment:
changed them to be in the same order as they are exported in insight.index.ts for readability